app/components/view-expandable-section/macro.njk

```diff
@@ -1,10 +1,10 @@
 {% from 'components/details/macro.njk' import details %}
 
 {% macro viewExpandableSection(params) %}
-  <div data-test-id="view-section-{{ params.sectionId }}">
+  <div data-test-id="{{ params.dataTestIdIdentifier }}" class="bc-c-expandable-section {{ params.classes }}">
```

Comment:
where is this class coming from `bc-c-expandable-section` if we don't a base style don't think we should add this perhaps?